docs/DeveloperGuide.md

```diff
@@ -254,21 +174,25 @@ in the Activity Diagram. This is a known limitation of PlantUML.</div>
 
 Given below is an example usage scenario of how an applicant is added, and how the operation is handled by LinkedOUT:
 
-Step 1: The user enters a valid add command, for example: `add n/Bob p/99999999 e/bob@example.com j/Data Analyst r/Interview s/Pandas s/Python s/Java`. For each command
+1. The user enters a valid add command, for example: `add n/Bob p/99999999 e/bob@example.com j/Data Analyst r/Interview s/Pandas s/Python s/Java`. For each command
 `LogicManager#execute()` is invoked, which calls `LinkedoutParser#parseCommand()` to separate the command word `add` and the argument
 `n/Bob p/99999999 e/bob@example.com j/Data Analyst r/Interview s/Pandas s/Python s/Java`
 
-Step 2: Upon identifying the add command, `AddCommandParser` is instantiated and uses `AddCommandParser#parse()` to
+
+2. Upon identifying the add command, `AddCommandParser` is instantiated and uses `AddCommandParser#parse()` to
 map the various prefixes to the attributes: (e.g `n/` to `Bob`, `p/` to `99999999`)
 
-Step 3: `AddCommandParser#arePrefixesPresent()` is called to ensure all the mandatory prefixes have been inputted by the user. After which
+
+3. `AddCommandParser#arePrefixesPresent()` is called to ensure all the mandatory prefixes have been inputted by the user. After which
 `AddCommandParser#parse()` creates the new `Applicant`
 
-Step 4. `AddCommandParser#parse()` then initializes an `AddCommand` with the new `Applicant` as an argument. `AddCommand#execute()`
+
+4. `AddCommandParser#parse()` then initializes an `AddCommand` with the new `Applicant` as an argument. `AddCommand#execute()`
 is then called, which calls `Model#hasApplicant()` to ensure that the new `Applicant` is not a duplicate of any existing applicant in the
 `LinkedOUT`. upon completion of the check, `Model#addApplicant()` to add the new applicant in the `LinkedOUT`.
```

Comment:
not sure if this will be change by goel later but `LinkedOUT` should be LinkedOUT instead

Reply:
Thanks for the spot, updated!